libfwupdplugin/fu-security-attrs.c

```diff
@@ -157,12 +157,17 @@ fu_security_attrs_remove_all(FuSecurityAttrs *self)
  *
  * Calculates the HSI string from the appended attributes.
  *
+ * If hsi_uint_ret is NULL, the guint type hsi will not be returned.
+ *
  * Returns: (transfer full): a string or %NULL
  *
  * Since: 1.5.0
+ *
  **/
 gchar *
-fu_security_attrs_calculate_hsi(FuSecurityAttrs *self, FuSecurityAttrsFlags flags)
+fu_security_attrs_calculate_hsi(FuSecurityAttrs *self,
+				FuSecurityAttrsFlags flags,
+				guint *hsi_guint_ret)
 {
 	guint hsi_number = 0;
```

Comment:
rename this `hsi_number_tmp` and use `hsi_number` in the function header.

Reply:
OK. I had modified all of them.